src/main/features/dashboard/views/claimant.njk

```diff
@@ -51,7 +51,7 @@
           </li>
           <li>
             <strong class="bold">{{ t('Claim details:') }}</strong>
-            <span class="block"><a href="{{ receiptUri }}">{{ t('Download claim') }}</a></span>
+            <span class="block"><a href="{{ ResponsePaths.claimDetailsPage.evaluateUri({ externalId: claim.externalId }) }}">{{ t('View claim') }}</a></span>
```

Comment:
That does not work for me, I am getting 'Forbidden error' because `IsDefendantInCaseGuard` is used for that path.

Also please remove `receiptUri` from `features/dashboard/routes/claimant.ts`.